aten/src/ATen/native/cuda/AdaptiveAveragePooling.cu

```diff
@@ -589,7 +598,8 @@ namespace {
       case at::MemoryFormat::ChannelsLast: {
         // special case for tensor memory format in channels_last
         TORCH_CHECK(input.ndimension() == 4,
-          "non-empty 4D (batch mode) tensor expected for input with channels_last layout");
+                    "Expected non-empty vector or matrix with optional 0-dim batch size, but got",
```

Comment:
Same question, if you want to add a space after `got` here for readability reasons?